README.md

```diff
@@ -30,6 +30,21 @@ e.g. on a Linux system you might want to build for local use like this:
 
 This will put an IrrlichtMtTargets.cmake file into the cmake directory in the current build directory, and it can then be imported from another project by pointing `find_package()` to the build directory, or by setting the `CMAKE_PREFIX_PATH` variable to that same path.
 
+on Windows system:
+
+It is highly recommended to use vcpkg as package manager.
+
+After you successfully built vcpkg you can easily install the required libraries:
+
+	vcpkg install zlib libjpeg-turbo libpng opengl-registry --triplet x64-windows
+	
+Run the following script in PowerShell:
+
+	git clone https://github.com/minetest/irrlicht
+	cd irrlicht
+	cmake -B build -G "Visual Studio 17 2022" -A "Win64" -DCMAKE_TOOLCHAIN_FILE=D:/vcpkg/scripts/buildsystems/vcpkg.cmake -DBUILD_SHARED_LIBS=OFF
```

Comment:
The D: location is not universal. If this cannot be made generic, it might be best to provide instructions on how to locate this cmake file.

Reply:
@SmallJoker, there is an example on vcpkg documentation
https://vcpkg.io/en/docs/users/buildsystems/cmake-integration.html#cmake_toolchain_file
Is it suitable in our case?

```
cmake ../my/project -DCMAKE_TOOLCHAIN_FILE=[vcpkg-root]/scripts/buildsystems/vcpkg.cmake
```

Reply:
Yes, that would be an improvement for sure.

Reply:
Updated README.md